### PR TITLE
Enable displaying BTC in mBTC, Bits, Satoshi units

### PR DESCRIPTION
**Pull request by rnevet**
1. Enable displaying BTC in mBTC, Bits, Satoshi units.
2. Correctly display summary if output currency isn't BTC.
3. Allow exiting bot loop from sleep after error.

Since there is a JS update included make sure to fully refresh the HTML page.

## Description
Implements https://github.com/Mikadily/poloniexlendingbot/issues/197
Fixes https://github.com/Mikadily/poloniexlendingbot/issues/196

## TESTING STAGE
In Testing

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] **I have read CONTRIBUTING.md**
- [x] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [x] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [N/A] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [x] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

### CODE REVIEW

File: www/lendingbot.js [hunk 1]
```diff
@@ -7,10 +7,17 @@ var Day = new Timespan("Day",1);
 var Week = new Timespan("Week",7);
 var Month = new Timespan("Month",30);
 var timespans = [Month, Week, Day, Hour];
-var Coin_Val = 1.00000000;
-var Coin = "BTC";
+var summeryCoinRate = 1.00000000;
+var summeryCoin = "BTC";
```
Comment (Evanito) [inline www/lendingbot.js]: it's "summary"

Comment (rnevet) [inline www/lendingbot.js]: true, it's -5 degrees here... I need summer! :)

I also need to fix the USDT calculation which I broke.

**Comment by Evanito** [comment]
I suggest we repoint this to master, really need to merge #174

**Comment by rnevet** [comment]
What's prevents you from merging, you don't need to wait for this PR.

**Comment by rnevet** [comment]
@Evanito please review

File: docs/configuration.rst [hunk 2]
```diff
@@ -304,3 +304,7 @@ You can pass options to statistics page by adding them to URL. Eg, ``http://loca
 - Allowed values: ``lentperc``, ``onlyfee``.
 - Default value: ``lentperc``.
 - ``onlyfee`` calculates effective rate without considering coin lent percentage.
+
+``displayUnit`` controls BTC unit output
+- Allowed values: ``BTC``, ``mBTC``, ``uBTC``, ``Satoshi``.
```
Comment (Evanito) [inline docs/configuration.rst]: Consider using `Bit` instead of `uBTC`, since it is a more recognised standard

File: lendingbot.py [hunk 3]
```diff
@@ -62,44 +62,47 @@
...
+                print "Unhandled error, please open a Github issue so we can fix it!"
+            sys.stdout.flush()
+            time.sleep(Lending.get_sleep_time())
+            pass
+except KeyboardInterrupt:
+    if web_server_enabled:
+        WebServer.stop_web_server()
+    log.log('bye')
+    print 'bye'
+    os._exit(0)  # Ad-hoc solution in place of 'exit(0)' TODO: Find out why non-daemon thread(s) are hanging on exit
```
Comment (Evanito) [inline lendingbot.py]: I see where the mistake was made, does `exit()` not work in a loop?
If your solution works, we do not need `os._exit(0)` (force exit), just `exit(0)`

**Comment by Evanito** [comment]
I approve this PR as is, but suggest the changes above. Also Codacy has some quick changes it wants done.

File: lendingbot.py (hunk 3 above)
Comment (rnevet) [inline lendingbot.py]: Exit works in a loop, the issue I fixed was related to using CTRL C during the default error handling sleep - which would then not exit.

**Pull request merged**